### PR TITLE
Create chapters_counter method in tutorials_controller.rb #180

### PR DESCRIPTION
**Pull request by bbiehl**

### CODE REVIEW

**Comment by bbiehl** [comment]
I can't remember how to do this and am unable to find a useful google search.

File: app/controllers/tutorials_controller.rb [hunk 1]
```diff
@@ -34,6 +34,18 @@ def view
     render "view"
   end
 
+  def chapters_arr
+    @chapters = Chapter.where({tutorial_id: @tutorial.id})
+  end
+
+  def chapter_counter
```
Comment (sumeetjain) [inline app/controllers/tutorials_controller.rb]: Remove unused methods (tutorialscontroller#chapter_counter)

File: app/controllers/tutorials_controller.rb [hunk 2]
```diff
@@ -34,6 +34,18 @@ def view
...
+    i = 1
+    while i <= chapters_arr.length
```
Comment (sumeetjain) [inline app/controllers/tutorials_controller.rb]: Refers to i more than self (maybe move it to another class?) (FeatureEnvy)

File: app/controllers/tutorials_controller.rb [hunk 3]
```diff
@@ -34,6 +34,18 @@ def view
     render "view"
   end
 
+  def chapters_arr
+    @chapters = Chapter.where({tutorial_id: @tutorial.id})
+  end
+
+  def chapter_counter
+    i = 1
```
Comment (sumeetjain) [inline app/controllers/tutorials_controller.rb]: Has the variable name 'i' (UncommunicativeVariableName)

File: app/controllers/tutorials_controller.rb [hunk 4]
```diff
@@ -34,6 +34,18 @@ def view
...
+    while i <= chapters_arr.length
+      return i
+      i += 1
```
Comment (sumeetjain) [inline app/controllers/tutorials_controller.rb]: Unreachable code detected.

**Comment by bbiehl** [comment]
Actually didn't require a chapters_counter method at all.  We simply did the counting on the view/tutorials.html.erb via including an @counter variable in our view method in tutorials_controller.rb

**Pull request merged**